tests/playwright/jest.e2e.config.js

```diff
@@ -1,4 +1,15 @@
+const path = require( 'path' );
+
 module.exports = {
+	globals: {
+		baseUrl:
+			( process.env.MW_SERVER || 'http://localhost:8080' ) +
+			( process.env.MW_SCRIPT_PATH || '/w' ),
+		display: process.env.DISPLAY,
```

Comment:
why is there a separate PR for test within a same file? I think we agreed to create multiple issues for each test but push test for a particular file in the same `PR`

Reply:
@und3fined-v01d, this should be WIP. Sorry about the confusion.